### ISSUE TITLE
Need new pictures

### ISSUE THREAD

**Issue by cmcrawford2** [cmcrawford2/memory-game#2]
The pictures that I use are directly scanned from the 1968 game. This probably violates copyright laws. We need a new set of pictures from free use sources. Any number will do. When we have 54 pictures, we can close this issue and switch over to those pictures. Pictures should go in a new directory for now.

**Comment by ClJarvis** [comment]
Is there any theme you want for the images? 

Or a theme for a subset of images? I'd like to draw a couple things.

**Comment by cmcrawford2** [comment]
Yes! I only have the capability to post pics that are 80px by 80px. So whatever you draw, you have to shrink it down to that before you upload it to here. Aside from that, they should be pictures of one thing with solid or no background, suitable for children. I'd like to keep the look and feel of the game the same as the original.

**Comment by cmcrawford2** [comment]
Hi! I've added an empty (except for .gitkeep) folder called new_cards to store new cards. You should update if you forked already. They should have the same name as the ones in the public folder. card1.jpeg, card2.jpeg, etc.
